docs(styleguide): FRMW-289 HTML Indentation Standard

ssalinas24 opened the pull request:
JIRA:
https://jira.rax.io/browse/FRMW-289
and
https://jira.rax.io/browse/FRMW-291

- [x] Dev LGTM
- [x] Dev LGTM
- [x] Dev LGTM

ssalinas24 commented on guides/html-styleguide.md:
```diff
@@ -0,0 +1,32 @@
+
+## Basic Rules:
+
+1. **Use Two Space Indentations**
```
@CITguy I put my bullet list mark down (line 9 - 32) in the editor at http://jester.surge.sh/ as suggested by Freddy. When I have it on there it looks the way I want it to. However, within my local it loses a few line breaks that I would like it to have and doesn't look the same. Please let me know if I can improve on something.

CITguy replied:
Try adding a blank line between the list item and the extra content.

100stacks commented on guides/html-styleguide.md:
````diff
@@ -0,0 +1,32 @@
+   </ul>
+   ```
+
+2. **HTML Max Line Length**
````
@ssalinas24 do we need to mention an exception for URLs that may extend pass 120 characters, since a hard line break will break the `href`?

LGTM

CITguy replied:
Good suggestion.

ssalinas24 replied:
@100stacks Good catch. I had originally thought about it, but forgot to include after i got distracted with the formatting. I will update.

CITguy replied:
Leave an empty line after this line to get numbered list items to display consistently.

```markdown
2. **HTML Max Line Length**

    The maximum line length ...
```

CITguy commented on guides/html-styleguide.md:
```diff
@@ -0,0 +1,32 @@
+
+## Acknowledgements:
+
+Most of these standards are based off of the guidelines provided by [Google](https://google.github.io/styleguide/htmlcssguide.xml#Indentation), and current developer feedback. Reading up on these materials will give you a much greater understanding of why these rules were chosen.
```
Ironically, this breaks the 120 character limit rule being added.

CITguy commented on guides/html-styleguide.md:
````diff
@@ -0,0 +1,32 @@
+   ```
+
+   *Scenario #2*
+   This is an example of using the proper indentation with nested elements. Each child element is nested and indented two spaces from its parent element.
````
Ironically, this breaks the 120 character limit rule being added.

CITguy commented on guides/html-styleguide.md:
````diff
@@ -0,0 +1,32 @@
+   ```
+
+2. **HTML Max Line Length**
+   The maximum line length for HTML files should be 120 characters. Review line lengths during coding of HTML files, and break to a new line if it exceeds 120 characters. Hyperlinks are an exceptions to this rule. Hyperlinks can exceed the 120 character limit and must not be broken or separated.
````
Ironically, this breaks the 120 character limit rule being added.

100stacks replied:
@CITguy @ssalinas24 I ran into the same issue of long URLs breaking the 120 character limit rule when updating the CSS Styleguide.  The referenced links are very beneficial for providing context within `EncoreUI`.

ssalinas24 commented:
@CITguy I see what you mean on ironically being over 120 characters. LOL I didn't snap that I had to manually break some of those lines in the .md file or they were actually wrapping as one continuous line. I have made the corrections and squashed.

bry4n commented:
LGTM

CITguy commented on guides/html-styleguide.md:
```diff
@@ -0,0 +1,40 @@
+
+## Basic Rules:
+
+1. **Use Two Space Indentations**
```
Leave an empty line after this line to get numbered list items to display consistently.

```markdown
1. **Use Two Space Indentations**

    The framework is indenting ...
```

ssalinas24 commented:
@CITguy I have formatted this to use headers instead of bullet points as we discussed. I updated and squashed. Let me know if this needs any additional changes.

CITguy commented:
LGTM
